code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -212,3 +213,15 @@
 
 /obj/item/restraints/handcuffs/cable/zipties/used/attack()
 	return
+
+/obj/item/restraints/handcuffs/twimsts
+	name = "twimsts cuffs"
+	desc = "Liquorice twist candy made into cable cuffs, tasty but it can't actually hold anyone."
+	icon_state = "cablecuff"
+	item_state = "cablecuff"
+	cuffed_state = "cablecuff"
+	belt_icon = "cablecuff"
+	color = "#E31818"
+	throwforce = 0
+	breakouttime = 0
+	cuffsound = 'sound/weapons/cablecuff.ogg'
```

Comment:
Might be better to make this a cablecuff subtype so you don't have to copy most of this?

Reply:
It was initially a subtype.

Reply:
Whoops I see, you can resolve this